src/lib/decorators.ts

```diff
@@ -63,13 +63,35 @@ export const property = (options?: PropertyDeclaration) => (proto: Object,
 /**
  * A property decorator that converts a class property into a getter that
  * executes a querySelector on the element's renderRoot.
+ *
+ * __Usage__
```

Comment:
We don't have `__Usage__` in any other jsdocs here. We should use the `@example` tag: http://usejsdoc.org/tags-example.html